handlers/admin/data-contributions.js

```diff
@@ -16,23 +20,30 @@ function editDataContribution(request, reply) {
       request.yar.flash('success', 'Data Contribution updated successfully.');
       return reply.redirect('/admin/data-contributions');
     })
-    .catch(() => {
-      // FIXME: Log error
+    .catch((error) => {
+      console.error(error.detail); // eslint-disable-line no-console
```

Comment:
Those logs from console are already sent to our Papertrail account, so all's good. In the future, I'd like to enable Sentry, as I find it more useful than just the logs.